source/js/apiHelpers.js

```diff
@@ -22,15 +22,23 @@ export async function getDetailedRecipeInfoBulk(idsToFetch) {
       resolve([]);
     } else {
       const idsFormatted = idsToFetch.join(',');
-      fetch(`https://${HOST}/recipes/informationBulk?&ids=${idsFormatted}`, {
+      fetch(`https://${HOST}/recipes/informationBulk?&ids=${idsFormatted}&includeNutrition=true`, {
         method: 'GET',
         headers: {
           'x-rapidapi-host': HOST,
           'x-rapidapi-key': API_KEY,
         },
       })
-        .then((response) => {
-          resolve(response.json());
+        .then((response) => response.json())
+        .then((data) => {
+          data.forEach((recipe) => {
+            const { nutrients } = recipe.nutrition;
+            // eslint-disable-next-line no-param-reassign
+            delete recipe.nutrition;
+            // eslint-disable-next-line no-param-reassign
+            recipe.nutrients = nutrients;
+          });
+          resolve(data);
```

Comment:
try not to ignore linting errors if you can fix them. most of them are valid comments when you do something wrong. 

this block might work but its not really a good practice to modify a function parameter. Instead I would copy it into a new variable and return that instead (you need to map through the elements and modify them instead of a for loop)

```
.then((data) => {
  resolve(data.map((recipe) => {
    const recipeInfo = recipe;
    const { nutrients } = recipe.nutrition;
    delete recipeInfo.nutrition;
    recipeInfo.nutrients = nutrients;
    return recipeInfo;
  }));
})
```
please verify that this code does the same thing you're trying to accomplish and replace it if you think its good so we don't have to bypass linting warnings.

Reply:
lol I think this way might also work as a fun one-liner if you want to learn about object destructuring but you really don't need to use it
```
.then((data) => {
  resolve(
    data.map(({ nutrition, ...rest }) => ({ nutrients: nutrition.nutrients, ...rest })),
  );
})
```

Reply:
Ahh okay ill do it this way